common/static/common/js/components/views/paging_header.js

```diff
@@ -1,113 +1,64 @@
-define(["underscore", "backbone", "gettext", "text!common/templates/components/paging-header.underscore"],
```

Comment:
As mentioned before, this is really a completely different header implementation. I'd move this file back to cms, and then create a new file here.